docs/eks-installation.md

```diff
@@ -117,6 +121,127 @@ Loading locally built `antrea/nephe` images to EKS cluster.
 ~/terraform/eks destroy
 ```
 
+## Setup AWS IAM Role Based Access [Optional]
+
+In addition to using AWS credentials for access, Nephe provides support for IAM
+role-based access, offering an alternative method to grant access and
+permissions. IAM role-based access allows for fine-grained control, simplified
+access management and improved security by eliminating the need to manage
+individual AWS credentials. However, the limitation of IAM role-based access is
+that it can only be used when Nephe is deployed on an EKS cluster.
+
+This documentation section will guide you through the setup process in AWS for
+IAM role-based access for Nephe.
+
+### Create an IAM Policy
+
+Navigate to the IAM dashboard on AWS console and select policies.
+Create a policy with the following permissions.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Action": [
+      "ec2:AuthorizeSecurityGroupEgress",
+      "ec2:AuthorizeSecurityGroupIngress",
+      "ec2:CreateSecurityGroup",
+      "ec2:CreateTags",
+      "ec2:DeleteSecurityGroup",
+      "ec2:DeleteTags",
+      "ec2:DescribeInstances",
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:DescribeSecurityGroups",
+      "ec2:DescribeVpcs",
+      "ec2:DetachInternetGateway",
```

Comment:
Why is this permission needed?

Reply:
Not sure, maybe something related to NICs? These permissions were tested using integration tests back then, and missing any of these causes CI to fail.

Reply:
Lets not include IAM role permission which are required for terraform. I will create a new one which is just used for Nephe.. Lets discuss offline